bin/propolis-server/src/lib/stats.rs

```diff
@@ -169,13 +167,13 @@ pub async fn start_oximeter_server(
     );
     let config = Config {
         server_info,
-        registration_address,
-        dropshot: dropshot_config,
+        registration_address: Some(registration_address),
+        request_body_max_bytes: 2048,
```

Comment:
Could we get some documentation for this magic number?  (And perhaps toss it in a `const`, while we're here?)

Reply:
FWIW, it looks like it was also a magic number previously, and this commit just moved it: https://github.com/oxidecomputer/propolis/pull/689/files#diff-b0ffb61cea8161225fe6897fd730efc8c2521123536783a990abe00d37fccbb6L152

But, I agree that a constant would be nice.